frontstage/views/surveys/access_survey.py

```diff
@@ -31,13 +31,11 @@ def access_survey(session):
         )
         eq_version = collection_exercise["eqVersion"]
         if eq_version == "v3":
```

Comment:
Both of the blocks here are identical?  Just remove this v3 if/else statement and just have the 1 redirect statement